addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -488,27 +497,78 @@ def test_message_process_alias_config_bounced_to(self):
         self.assertIn(bounce_message_with_alias, self._mails[0].get('body'))
 
     @mute_logger('odoo.addons.mail.models.mail_thread', 'odoo.models')
-    def test_message_process_alias_defaults(self):
+    @patch('odoo.addons.mail.models.mail_alias.Alias._on_message_alias_config_error')
+    def test_message_process_alias_defaults(self, _on_message_alias_config_error):
         """ Test alias defaults and inner values """
-        self.alias.write({
+        container_custom_value = self.env['mail.test.container'].create({})
+        self.alias_valid.write({
             'alias_user_id': self.user_employee.id,
-            'alias_defaults': "{'custom_field': 'defaults_custom'}"
+            'alias_defaults': f"{{'container_id': {container_custom_value.id}}}",
```

Comment:
Actually, why changing this test ? It seems it could work with old values (just add tests on alias_status), isn't it ?

Reply:
Actually, the idea was to change it to something similar as in the test where it fails so we show it works with a reference in the alias defaults and it fails when that reference is dangling. 
But it is also great to move the definition alias_valid from the setup to the test where it is used. So I have followed your advice and changed a bit the test about the invalid status: by testing first that it works when the reference is not dangling then testing that it fails otherwise.